README.md

```diff
@@ -74,10 +74,29 @@ DNADesign\SecurityHeaders\SecurityHeadersExtension:
         - '''self'''
 ```
 
+## Environment variables
+
+Environment varibles can be used by wrapping with `
+```yml
+DNADesign\SecurityHeaders\SecurityHeadersExtension:
+  whitelist:
+    connect-src:
+      data:
+        - "'self'"
+        - "`ENVIRONMENT_VARIABLE_EXAMPLE`"
```

Comment:
Why do we need this? I don't think we want to manage security headers using env vars.